<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TMOGucKaynagiListeFormu</class>
 <widget class="QDialog" name="TMOGucKaynagiListeFormu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>992</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Güç Kaynağı Liste Formu</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../Kaynaklar.qrc">
    <normaloff>:/resimler/guckaynagi.jpg</normaloff>
    <normalon>:/resimler/guckaynagi.jpg</normalon>:/resimler/guckaynagi.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">TMOGucKaynagiListeFormu{

	
	background-image: url(:/resimler/guckaynagi.jpg);
}

QLabel {
   font:Bold; 
   color: #ffffff;
}
QGroupBox {
    font:Bold;
color: #ffffff;
}
QPushButton{font:Bold;}
QRadioButton{
    font:Bold;
color: #ffffff;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="0" column="0">
    <widget class="QGroupBox" name="GPAramaBolumu">
     <property name="title">
      <string>GÜÇ KAYNAĞI ARAMA BÖLÜMÜ</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QGroupBox" name="GPParcaMarkasi">
        <property name="title">
         <string>Parça Markası</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="LPMarkasiDeger">
           <property name="text">
            <string>Değer:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="LEPMarkasiDeger"/>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QGroupBox" name="GPPMarkasiFiltreTuru">
           <property name="title">
            <string>Filtre Türü</string>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QRadioButton" name="RBPMarkasiFTIleBaslayan">
              <property name="text">
               <string>İle Başlayan</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QRadioButton" name="RBPMarkasiFTIleBiten">
              <property name="text">
               <string>İle Biten</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QRadioButton" name="RBPMarkasiFTIceren">
              <property name="text">
               <string>İçeren</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="PBAra">
        <property name="text">
         <string>ARA</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QGroupBox" name="GPParcaModeli">
        <property name="title">
         <string>Parça Modeli</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="LPModeliDeger">
           <property name="text">
            <string>Değer:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="LEPModeliDeger"/>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QGroupBox" name="GPPModeliFiltreTuru">
           <property name="title">
            <string>Filtre Türü</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QRadioButton" name="RBPModeliFTIleBaslayan">
              <property name="text">
               <string>İle Başlayan</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QRadioButton" name="RBPModeliIFTIleBiten">
              <property name="text">
               <string>İle Biten</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QRadioButton" name="RBPModeliFTIceren">
              <property name="text">
               <string>İçeren</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="PBKapat">
        <property name="text">
         <string>KAPAT</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTableWidget" name="TWTablo"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../Kaynaklar.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>PBAra</sender>
   <signal>clicked()</signal>
   <receiver>TMOGucKaynagiListeFormu</receiver>
   <slot>aramaYap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>861</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>987</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PBKapat</sender>
   <signal>clicked()</signal>
   <receiver>TMOGucKaynagiListeFormu</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>947</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>987</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RBPModeliFTIceren</sender>
   <signal>clicked()</signal>
   <receiver>TMOGucKaynagiListeFormu</receiver>
   <slot>aramaYap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>709</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>989</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RBPModeliIFTIleBiten</sender>
   <signal>clicked()</signal>
   <receiver>TMOGucKaynagiListeFormu</receiver>
   <slot>aramaYap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>582</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>987</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RBPModeliFTIleBaslayan</sender>
   <signal>clicked()</signal>
   <receiver>TMOGucKaynagiListeFormu</receiver>
   <slot>aramaYap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>493</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>988</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LEPModeliDeger</sender>
   <signal>textChanged(QString)</signal>
   <receiver>TMOGucKaynagiListeFormu</receiver>
   <slot>aramaYap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>616</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>988</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LEPMarkasiDeger</sender>
   <signal>textChanged(QString)</signal>
   <receiver>TMOGucKaynagiListeFormu</receiver>
   <slot>aramaYap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RBPMarkasiFTIleBaslayan</sender>
   <signal>clicked()</signal>
   <receiver>TMOGucKaynagiListeFormu</receiver>
   <slot>aramaYap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RBPMarkasiFTIleBiten</sender>
   <signal>clicked()</signal>
   <receiver>TMOGucKaynagiListeFormu</receiver>
   <slot>aramaYap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RBPMarkasiFTIceren</sender>
   <signal>clicked()</signal>
   <receiver>TMOGucKaynagiListeFormu</receiver>
   <slot>aramaYap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>aramaYap()</slot>
 </slots>
</ui>
